pkg/api/bytes.go

```diff
@@ -52,7 +61,16 @@ func (s *Service) bytesUploadHandler(w http.ResponseWriter, r *http.Request) {
 	if err != nil {
 		logger.Debug("bytes upload: get or create tag failed", "error", err)
 		logger.Error(nil, "bytes upload: get or create tag failed")
-		jsonhttp.InternalServerError(w, "cannot get or create tag")
+		switch {
+		case errors.Is(err, tags.ErrExists):
+			jsonhttp.Conflict(w, "bytes upload: conflict with current state of resource")
+		case errors.Is(err, errCannotParse):
```

Comment:
This case will not work. Also, edit the responses to be consistent with the others.